src/constants/translations/en/common.json

```diff
@@ -69,7 +69,8 @@
   "errorMessages": {
     "emailValid": "Email should be of the following format: “local-part@domain.com”",
     "passwordLength": "Password cannot be shorter than 8 and longer than 25 characters",
-    "passwordValid": "Password must contain at least one alphabetic and one numeric character",
+    "passwordAlphabeticAndNumeric": "Password must contain at least one alphabetic and one numeric character",
+    "passwordValidSymbols": "Password must contain only alphabetic, numeric and special characters",
```

Comment:
Did yout provide corresponding translations in ua language?

Reply:
Regarding translation, there are no corresponding ua translations for many messages. There is no even `errorMessages` object in `translations/ua/common.json`. So I thought it would be better to make separate pr with all missing ua translations.

Reply:
Hope to see new pull request